packages/rocketchat-importer-slack/server/importer.js

```diff
@@ -121,6 +121,230 @@ export class SlackImporter extends Base {
 		return new Selection(this.name, selectionUsers, selectionChannels, selectionMessages);
 	}
 
+	performUserImport(user, startedByUserId) {
+		if (!user.do_import) {
+			return;
+		}
+
+		Meteor.runAsUser(startedByUserId, () => {
+			const existantUser = RocketChat.models.Users.findOneByEmailAddress(user.profile.email) || RocketChat.models.Users.findOneByUsername(user.name);
+			if (existantUser) {
+				user.rocketId = existantUser._id;
+				RocketChat.models.Users.update({ _id: user.rocketId }, { $addToSet: { importIds: user.id } });
+				this.userTags.push({
+					slack: `<@${ user.id }>`,
+					slackLong: `<@${ user.id }|${ user.name }>`,
+					rocket: `@${ existantUser.username }`,
+				});
+			} else {
+				const userId = user.profile.email ? Accounts.createUser({ email: user.profile.email, password: Date.now() + user.name + user.profile.email.toUpperCase() }) : Accounts.createUser({ username: user.name, password: Date.now() + user.name, joinDefaultChannelsSilenced: true });
+				Meteor.runAsUser(userId, () => {
+					Meteor.call('setUsername', user.name, { joinDefaultChannelsSilenced: true });
+
+					const url = user.profile.image_original || user.profile.image_512;
+					try {
+						Meteor.call('setAvatarFromService', url, undefined, 'url');
+					} catch (error) {
+						this.logger.warn(`Failed to set ${ user.name }'s avatar from url ${ url }`);
+						console.log(`Failed to set ${ user.name }'s avatar from url ${ url }`);
+					}
+
+					// Slack's is -18000 which translates to Rocket.Chat's after dividing by 3600
+					if (user.tz_offset) {
+						Meteor.call('userSetUtcOffset', user.tz_offset / 3600);
+					}
+				});
+
+				RocketChat.models.Users.update({ _id: userId }, { $addToSet: { importIds: user.id } });
+
+				if (user.profile.real_name) {
+					RocketChat.models.Users.setName(userId, user.profile.real_name);
+				}
+
+				// Deleted users are 'inactive' users in Rocket.Chat
+				if (user.deleted) {
+					Meteor.call('setUserActiveStatus', userId, false);
+				}
+
+				user.rocketId = userId;
+				this.userTags.push({
+					slack: `<@${ user.id }>`,
+					slackLong: `<@${ user.id }|${ user.name }>`,
+					rocket: `@${ user.name }`,
+				});
+			}
+
+			this.addCountCompleted(1);
+		});
+	}
+
+	processMessageSubType(message, room, msgDataDefaults, missedTypes) {
+		let msgObj;
+		const ignoreTypes = { bot_add: true, file_comment: true, file_mention: true };
+
+		let rocketUser = this.getRocketUser(message.user);
+		if (!rocketUser) {
+			rocketUser = RocketChat.models.Users.findOneById('rocket.cat', { fields: { username: 1, name: 1 } });
+		}
+
+		if (rocketUser) {
```

Comment:
instead of adding the rest code inside this `if` I suggest to change it to:
```javascript
if (!rocketUser) {
	return;
}

switch (message.subtype) {
	...
}
```